src/docs/Components/Tooltip/index.mdx

```diff
@@ -9,7 +9,7 @@ status:
 
 > A tooltip allows you to display further informations to the user when needed. It can also be used to explicitely specify a visual element like an icon.
 
---preview of the four tooltips--
```

Comment:
Just a little detail → 2px are missing for the tooltip height.
Can you improve the arrow like that we have in Sketch?

![tooltip-arrow](https://zupimages.net/up/20/29/92uw.png)

Reply:
@caqueste Your remarks have been taken into account. It's been corrected.